exasol_script_languages_container_ci_setup/cli/commands/start_test_release_build.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+
+import click
+
+from exasol_script_languages_container_ci_setup.cli.cli import cli
+from exasol_script_languages_container_ci_setup.cli.common import add_options
+from exasol_script_languages_container_ci_setup.cli.options.logging import logging_options, set_log_level
+from exasol_script_languages_container_ci_setup.lib.aws_access import AwsAccess
+from exasol_script_languages_container_ci_setup.cli.options.aws_options import aws_options
+from exasol_script_languages_container_ci_setup.lib.github_release_creator import GithubReleaseCreator
+from exasol_script_languages_container_ci_setup.lib.run_start_release_build import run_start_test_release_build
+
+
+@cli.command()
+@add_options(aws_options)
+@add_options(logging_options)
+@click.option('--project', type=str, required=True,
+              help="""The project name. Must be same name as used for the AWS CodeBuild release stack creation.""")
+@click.option('--repo-id', type=str, required=True,
```

Comment:
What is the repo-id, I don't see it in the deploy function

Reply:
Or, is this id of the repo for the GitHub api

Reply:
`repo-id` is for example: `exasol/script-languages`. For the deploy we use a similar parameter: `project-url`, which would be `https://github.com/exasol/script-languages` for the same project. The project-url we forward to AWS. The `repo-id` we use in PyGithub. Of course we could think of harmonizing the API and use the same parameter for both commands, as one can be constructed from the other.

Reply:
The id part of the parameter name was confusing. Renaming it to repo-name should be enough

Reply:
Done